src/target/page/TargetPage.tsx

```diff
@@ -1,12 +1,26 @@
 import React, { ReactNode, useEffect, useState } from 'react';
 import { Notification } from 'hds-react';
-import api, { Target } from '../../common/utils/api/api';
+import api, {
+  Target,
+  SourceLink,
+  SourceLinkTypes,
+} from '../../common/utils/api/api';
 import Collapse from '../../components/collapse/Collapse';
+import Link from '../../components/link/Link';
 import './TargetPage.scss';
 
 const hasText = (str: string | null | undefined): boolean =>
   str !== undefined && str !== null && str !== '';
 
+const getTargetServiceAdminUrl = (
+  links: SourceLink[] = []
+): string | undefined => {
+  const adminLink = links.find(
```

Comment:
Can we return the result of this find operation directly, or is it easier to create this const first and then return it in line 21?

Reply:
Let's see how does it work 👀